### PR TITLE
Multiple editable field should be real multiple

### LINKED ISSUE #5767: Multiple choses in editable 'choice' type in List fields
## Feature Request

I want to do something like this:

```php
$listMapper
    ->add('winner', 'choice', [
        'label' => 'Winner',
        'editable' => true,
        'choices' => [
            'jury' => 'jury',
            'voting' => 'voting',
        ],
        'multiple' => true,
    ]);
```

image

### PR DESCRIPTION
**Pull request by peter-gribanov**
## Subject

Multiple editable field should be real multiple.

image

Now a multiple field is shown as multiple but edit is not a multiple and new value will be overridden, not append.

image

image

image

Closes #5767

## Changelog

```markdown
### Added
- Multiple editable fields now is a real multiple.
```

### CODE REVIEW

File: src/Resources/views/CRUD/base_list_field.html.twig [hunk 1]
```diff
@@ -35,7 +35,13 @@ file that was distributed with this source code.
         </a>
     {% else %}
         {% set isEditable = field_description.options.editable is defined and field_description.options.editable and admin.hasAccess('edit', object) %}
-        {% set xEditableType = field_description.type|sonata_xeditable_type %}
+        {% if isEditable==true and field_description.options.multiple is defined and field_description.options.multiple==true and value is iterable %}
```
Comment (core23) [inline src/Resources/views/CRUD/base_list_field.html.twig]:
```suggestion
        {% if isEditable and field_description.options.multiple is defined and field_description.options.multiple==true and value is iterable %}
```

Comment (peter-gribanov) [reply src/Resources/views/CRUD/base_list_field.html.twig]: Maybe we should use a single coding style?
https://github.com/sonata-project/SonataAdminBundle/search?l=Twig&q=%3D%3Dtrue

Comment (peter-gribanov) [reply src/Resources/views/CRUD/base_list_field.html.twig]: For example, what about this condition?

https://github.com/sonata-project/SonataAdminBundle/blob/7619f20f25fea2432784c490539557a68424cf96/src/Resources/views/CRUD/list_choice.html.twig#L33

**Comment by greg0ire** [comment]
Please kindly squash your commits together. If you don't, we'll try to remember to do it for you but it's best if you save us this trouble.

How to do that?

1. `git rebase -i origin/3.x`, assuming `origin` is a git remote that points to this repository, and not your fork. If you're not sure what your remotes are, run `git remote -vvv`, there should be your fork and the holy/reference/base/origin/whatever-you-call-it repository.
2. A window will show up with many lines, replace `pick` with `fixup` on every line but the first one
3. Close your editor, git should do its magic, and you should end up with one commit
4. Use `git push --force` to overwrite what you already push. Don't forget the `--force` option otherwise git will try to merge both things together.

Also, please format your commit message according to our rules : 

- The commit message subject must be less than 50 characters and tell us *what* you did.
- The commit message body should tell us *why* you did it. It is optional but highly recommended.

Bad example :

```
Fixed bug #989 by removing call to defraculate()
```

Good example:

```
Remove call to defraculate()

Calling this function caused a bug because it interferes 
with calls to getSchmeckles().
Fixes #989
```

More details in CONTRIBUTING.md

**Comment by peter-gribanov** [comment]
@greg0ire i squash  commits

**Comment by greg0ire** [comment]
Thanks @peter-gribanov !

**Pull request merged**